toolkit/tools/internal/packagerepo/repocloner/rpmrepocloner/rpmrepocloner.go

```diff
@@ -69,9 +71,10 @@ const (
 
 // RpmRepoCloner represents an RPM repository cloner.
 type RpmRepoCloner struct {
-	chroot         *safechroot.Chroot
-	usePreviewRepo bool
-	cloneDir       string
+	chroot              *safechroot.Chroot
+	usePreviewRepo      bool
+	disableMarinerRepos bool
```

Comment:
We don't need `disableMarinerRepos` to be a member of this struct anymore, since we only need it inside `Initialize`, where we have it as an input variable and can further pass it into `initializeRepoDefinitions`.